build-system/eslint-rules/preact-preferred-props.js

```diff
@@ -19,8 +19,12 @@
 // <div tabindex="0" />
 // <path strokeLinecap />
 
+const propNameFn = 'propName';
+const propNameFnModule = '#preact/utils';
+
 const DOM_ATTRIBUTES = {
   className: 'class',
+  // TODO(wg-bento): Revert tabIndex with tabindex
   tabindex: 'tabIndex',
```

Comment:
It's a shame that we missed that we're doing `tabIndex` the wrong way around. We should fix this afterwards. It's a breaking change that has to be documented and communicated.